RELEASE.md

```diff
@@ -0,0 +1,13 @@
+# Release
+
+## Protos
+
+To release protobuf definitions:
+
+``` sh { name=buf-token }
+export BUF_TOKEN=Your buf token
+```
+
+``` sh { name=release-buf }
+cd ./internal/api && buf push
```

Comment:
I believe we have a make target for this, no?

Reply:
Oh yeah, my bad